### PR TITLE
Fullres power level reports 0mW for 50mW

### PR DESCRIPTION
**Pull request by CapnBry**
This fixes the oversight in the fullres OTA which tries to stuff 9 power levels into 3 bits. This resulted in 50mW being displayed at 0mW. This removes 0mW (CRSF power 0) from our possibilities to make room for 50mW (CRSF power 8).

### Details
Only was a problem on FullRes, which only has 3 bits for the power level. Wide switch mode has 6/7 bits, and Hybrid doesn't send it at all.

### Solution
constrain the uplink power to 3 bits 1-8 instead of 0-7, this means that 0mW will be displayed as 10mW, but we never switch to 0mW mode even when the radio is disabled (it is below MinPower). Not sure if the constrain is needed, since we can't be at 0 power, but I included it for safety. I can remove it if we think that's a waste.

### CODE REVIEW

File: src/lib/OTA/OTA.h [hunk 1]
```diff
@@ -92,7 +92,7 @@ typedef struct {
         struct {
             uint8_t packetType: 2,
                     telemetryStatus: 1,
-                    uplinkPower: 3, // PowerLevels_e
+                    uplinkPower: 3, // PowerLevels_e - 1 (1-8 is 0-7 in the air)
```
Comment (pkendall64) [inline src/lib/OTA/OTA.h:95]: So this really isn't PowerLevels_e then is it? The comment should be
`CRSF_power_level - 1 (1-8 is 0-7 in the air)`

Comment (CapnBry) [reply src/lib/OTA/OTA.h:95]: (embarrassed paw over face meme) you're right! I'll fix that and the test

**Comment by CapnBry** [comment]
@pkendall64 I wanted to write a test that tests all our power levels and makes sure they go through the air OK so we'd catch this error if it happens again, so I had to move `powerToCrsfPower()` to somewhere common. I put it in POWERMGNT.cpp, but if you can think of a better place for it? Should I put it in OTA? Nothing else there knows anything about our Power enum so that felt wrong :/

**Comment by pkendall64** [comment]
> @pkendall64 I wanted to write a test that tests all our power levels and makes sure they go through the air OK so we'd catch this error if it happens again, so I had to move `powerToCrsfPower()` to somewhere common. I put it in POWERMGNT.cpp, but if you can think of a better place for it? Should I put it in OTA? Nothing else there knows anything about our Power enum so that felt wrong :/

I think in the POWERMGNT.cpp is probably the best place. I've been thinking we need to sort the power management out because it's all over the place currently, but that's a 3.1+ thing.

**Review by pkendall64 — APPROVED** [review]

**Review by FOG-Yamato — APPROVED** [review]

**Pull request merged**